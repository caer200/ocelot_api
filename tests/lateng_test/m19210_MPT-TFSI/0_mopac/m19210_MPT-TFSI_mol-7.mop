CHARGES

 
H	11.311481 +1 2.891194 +1 14.320323 +1
C	12.106887 +1 2.902100 +1 14.839832 +1
C	12.396700 +1 1.825928 +1 15.701447 +1
C	12.967522 +1 3.967132 +1 14.747145 +1
C	13.559748 +1 1.920622 +1 16.492813 +1
N	11.554706 +1 0.724244 +1 15.767804 +1
H	12.770366 +1 4.671212 +1 14.140420 +1
C	14.124701 +1 4.040659 +1 15.522318 +1
C	14.407782 +1 3.031330 +1 16.391596 +1
S	14.078787 +1 0.679236 +1 17.562974 +1
C	11.592163 +1 -0.221696 +1 16.790407 +1
C	10.489183 +1 0.588218 +1 14.759509 +1
H	14.708590 +1 4.786382 +1 15.444701 +1
H	15.186479 +1 3.080604 +1 16.933505 +1
C	12.683060 +1 -0.316390 +1 17.683967 +1
C	10.531818 +1 -1.124076 +1 16.966767 +1
H	10.320336 +1 -0.361476 +1 14.588598 +1
H	9.669819 +1 1.011557 +1 15.091282 +1
H	10.768829 +1 1.025472 +1 13.928043 +1
C	12.703773 +1 -1.288956 +1 18.695371 +1
H	9.778736 +1 -1.082946 +1 16.389163 +1
C	10.572209 +1 -2.066563 +1 17.967333 +1
H	13.450555 +1 -1.348613 +1 19.279567 +1
C	11.653785 +1 -2.149002 +1 18.842188 +1
H	9.847901 +1 -2.673063 +1 18.064132 +1
H	11.660499 +1 -2.798694 +1 19.535084 +1
