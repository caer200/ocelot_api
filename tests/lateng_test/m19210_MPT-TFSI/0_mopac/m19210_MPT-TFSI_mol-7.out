 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 22:32:20 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H         11.31148100  *   2.89119400  *  14.32032300  *
     2       C         12.10688700  *   2.90210000  *  14.83983200  *
     3       C         12.39670000  *   1.82592800  *  15.70144700  *
     4       C         12.96752200  *   3.96713200  *  14.74714500  *
     5       C         13.55974800  *   1.92062200  *  16.49281300  *
     6       N         11.55470600  *   0.72424400  *  15.76780400  *
     7       H         12.77036600  *   4.67121200  *  14.14042000  *
     8       C         14.12470100  *   4.04065900  *  15.52231800  *
     9       C         14.40778200  *   3.03133000  *  16.39159600  *
    10       S         14.07878700  *   0.67923600  *  17.56297400  *
    11       C         11.59216300  *  -0.22169600  *  16.79040700  *
    12       C         10.48918300  *   0.58821800  *  14.75950900  *
    13       H         14.70859000  *   4.78638200  *  15.44470100  *
    14       H         15.18647900  *   3.08060400  *  16.93350500  *
    15       C         12.68306000  *  -0.31639000  *  17.68396700  *
    16       C         10.53181800  *  -1.12407600  *  16.96676700  *
    17       H         10.32033600  *  -0.36147600  *  14.58859800  *
    18       H          9.66981900  *   1.01155700  *  15.09128200  *
    19       H         10.76882900  *   1.02547200  *  13.92804300  *
    20       C         12.70377300  *  -1.28895600  *  18.69537100  *
    21       H          9.77873600  *  -1.08294600  *  16.38916300  *
    22       C         10.57220900  *  -2.06656300  *  17.96733300  *
    23       H         13.45055500  *  -1.34861300  *  19.27956700  *
    24       C         11.65378500  *  -2.14900200  *  18.84218800  *
    25       H          9.84790100  *  -2.67306300  *  18.06413200  *
    26       H         11.66049900  *  -2.79869400  *  19.53508400  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H         11.3115      2.8912     14.3203
     2         C         12.1069      2.9021     14.8398
     3         C         12.3967      1.8259     15.7014
     4         C         12.9675      3.9671     14.7471
     5         C         13.5597      1.9206     16.4928
     6         N         11.5547      0.7242     15.7678
     7         H         12.7704      4.6712     14.1404
     8         C         14.1247      4.0407     15.5223
     9         C         14.4078      3.0313     16.3916
    10         S         14.0788      0.6792     17.5630
    11         C         11.5922     -0.2217     16.7904
    12         C         10.4892      0.5882     14.7595
    13         H         14.7086      4.7864     15.4447
    14         H         15.1865      3.0806     16.9335
    15         C         12.6831     -0.3164     17.6840
    16         C         10.5318     -1.1241     16.9668
    17         H         10.3203     -0.3615     14.5886
    18         H          9.6698      1.0116     15.0913
    19         H         10.7688      1.0255     13.9280
    20         C         12.7038     -1.2890     18.6954
    21         H          9.7787     -1.0829     16.3892
    22         C         10.5722     -2.0666     17.9673
    23         H         13.4506     -1.3486     19.2796
    24         C         11.6538     -2.1490     18.8422
    25         H          9.8479     -2.6731     18.0641
    26         H         11.6605     -2.7987     19.5351

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C13 H11 N S  =    26 atoms



      MOLECULAR POINT GROUP   :   Cs  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
